test/unit/api/regular/shuffle_v2/idxm.cpp

```diff
@@ -780,6 +799,32 @@ TTS_CASE("is_slide_right")
   test(std::array {na_, na_, 1, 2}, -1);
 };
 
+TTS_CASE("is_slide_left2")
+{
+  auto test = [](auto _in, int reg_groups, int expected)
+  {
+    auto in     = to_idxs(_in);
+    auto actual = eve::detail::idxm::is_slide_left_2(in, reg_groups).value_or(-1);
+    TTS_EQUAL(expected, actual) << tts::as_string(in);
+  };
+
+  test(std::array {0, 1, 2, 3}, 4, 0); // not slide 2, result is unimportant
+  test(std::array {1, 2, 3, 4}, 4, 1);
+  test(std::array {1, 2, 3, 16}, 16, 1);
+  test(std::array {1, we_, 3, 4}, 4, 1);
+  test(std::array {1, we_, 3, 16}, 16, 1);
+  test(std::array {2, 3, 4, we_}, 4, 2);
+  test(std::array {2, 3, 16, we_}, 16, 2);
+  // {2, 3, we_, we_} - is not an important case because it's not slide2 at all.
```

Comment:
how so ?

Reply:
It's a one register shuffle. So it's not slide2

Reply:
OK